README.md

```diff
@@ -28,6 +28,15 @@ from media query to special class.
     background: black
   }
 }
+
+@media (prefers-color-scheme: light) {
```

Comment:
Let’s keep example simple.

But we can mentioned somewhere that we support both `light` and `dark` schemes.

Reply:
You are right, it looks redundant. I think the phrase _plugin to make switcher to force dark or light theme by copying styles from media query to special class_ has already described that.

Removed the example for the light scheme.